client/jetpack-connect/signup.js

```diff
@@ -242,12 +413,20 @@ export class JetpackSignup extends Component {
 	}
 }
 
-const connectComponent = connect( null, {
-	createAccount: createAccountAction,
-	createSocialAccount: createSocialAccountAction,
-	errorNotice: errorNoticeAction,
-	recordTracksEvent: recordTracksEventAction,
-	warningNotice: warningNoticeAction,
-} );
+const connectComponent = connect(
+	( state ) => ( {
+		requestError: getRequestError( state ),
```

Comment:
`requestError` looks like too generic name for the prop. How about `loginRequestError` or something even more specific?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/41798/commits/a7dc628ea10846b4bea6e5b0b4887b1b74ed9fb5